### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by 4812571**
Integrated Mayhem, a fuzz testing solution by ForAllSecure. Mayhem is targeting existing fuzz tests in the repository

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google Inc.
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Address the copyright, and add a package stage.

File: mayhem/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
```
Comment (xansec) [inline mayhem/build.sh:1]: No need for a build script for just one line.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by xansec — CHANGES_REQUESTED** [review]
These fuzz targets still need to be renamed, even if we are accepting OSS-Fuzz integrations. Please make the requested changes.

**Comment by 4812571** [comment]
The fuzz targets have been renamed.

**Comment by 4812571** [comment]
I fixed those up - the action is reporting as failed, but the run still works as normal - see https://github.com/mayhemheroes/relay/pull/1 for another case of this problem.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**